src/model/regional-collection.v1.yaml

```diff
@@ -0,0 +1,81 @@
+$schema: http://json-schema.org/draft-04/schema#
+title: Collection
+type: object
+allOf:
+  - $ref: ../snippets/regional-collection.v1.yaml
+  - properties:
+        image:
+            type: object
+            properties:
+                banner:
+                    $ref: ../misc/image.v1.yaml
+            required:
+              - banner
+        editors:
+            description: List of editors, ordered by name
```

Comment:
Is a different term needed here?

Reply:
When we present these in the Journal it will be under the heading of Local editors.